src/components/cap-case.js

```diff
@@ -363,15 +363,26 @@ export default class CapCase extends LitElement {
 		);
 	}
 
-	getDocketNumber() {
-		if (true) {
-			return html`
-				<span>&middot;</span>
-				<span class="docket-number">Docket Number</span>
-			`;
-		} else {
-			return nothing;
-		}
+	getYearFromDate(str) {
+		const date = new Date(str);
+		return date.getFullYear();
+	}
+
+	formatDate(dateStr) {
+		const date = new Date(dateStr);
+		const formatOptions = { timeZone: "UTC", year: "numeric" };
+		if (dateStr.length > 4) formatOptions.month = "short";
+		if (dateStr.length > 7) formatOptions.day = "numeric";
+
+		return date.toLocaleDateString(undefined, formatOptions);
+	}
+
+	createCitationsString(caseCitations) {
+		return caseCitations.map((citation) => citation.cite).join(", ");
+	}
+
+	createCaseHeaderHeader(caseMetadata) {
+		return `${caseMetadata.name_abbreviation}, ${this.createCitationsString(caseMetadata.citations)} (${this.getYearFromDate(caseMetadata.decision_date)})`;
 	}
```

Comment:
Curious because I am working on ENG-561, and this string is used as the page's `title` attribute too:

Do you know if we need to strip out `vendor` cites here, like [the Django app does](https://github.com/harvard-lil/capstone/blob/develop/capstone/capdb/models.py#L1119)?

Reply:
Ah, I am probably not the right person to answer this question. If `vendor` needs to be excluded, I can make the update. Let me confirm with Jack, I think he would know 🤔

Reply:
👍 
Potentially also worth noting that the Django app includes the check `if self.decision_date` when building this string, so I guess maybe it's empty sometimes.

Reply:
I added the decision date check, Becky.

Reply:
Great! May I recommend a little tweak? If you do:
```
return `${caseMetadata.name_abbreviation}, ${this.createCitationsString(caseMetadata.citations)} `${ decisionYear ? "(" + decisionYear + ")" : ""}`;
```
then when there is no decision year, it will look like "Full Citation" instead of "Full Citation ()"

Reply:
Pushed and cleaned up the html a bit